src/main/msp/msp_protocol_v2_inav.h

```diff
@@ -45,3 +45,6 @@
 #define MSP2_INAV_SELECT_BATTERY_PROFILE        0x2018
 
 #define MSP2_INAV_DEBUG                         0x2019
+
+#define MSP2_BLACKBOX_CONFIG                    0x201A
```

Comment:
What about configurator support for this one?

Reply:
It is needed, which is why it's marked as blocked. JS is not really my forte, but I'll look at it in the next few days. It makes old boards with 2MB flash useful again, especially on fixed wing.